### PR TITLE
Add city selection to start screen

### PR DESCRIPTION
**Pull request by The0mikkel**
This update refactors the start screen, as well as it adds the city select functionality to the start screen. This solves #11.

It is important to note, that the stack navigation has been updated, and the choosen location is passed along as an id in the stack navigation.  
Likewise, styling is also refactored out in separate directory.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "react-dom": "18.2.0",
     "react-native": "0.72.5",
     "react-native-pager-view": "6.2.0",
-    "react-native-screens": "^3.25.0",
+    "react-native-screens": "~3.22.0",
```
Comment (jakob753951) [inline package.json:26]: hvorfor downgrader vi her?

**Review by jakob753951 — APPROVED** [review]
LGTM

**Review by MGertz — APPROVED** [review]
looks good to me.

File: package.json (hunk 1 above)
Comment (The0mikkel) [reply package.json:26]: Expo stuff :|

Comment (jakob753951) [reply package.json:26]: fair

**Review by JfriisJ — APPROVED** [review]
LGTM

**Pull request merged**